packages/desktop-client/src/components/manager/ConfigServer.js

```diff
@@ -1,25 +1,41 @@
-import React, { useState, useEffect, useCallback } from 'react';
+import React, { useState, useEffect } from 'react';
 import { useDispatch } from 'react-redux';
 import { useHistory } from 'react-router-dom';
-import * as actions from 'loot-core/src/client/actions';
 import {
   View,
   Text,
   Button,
-  ButtonWithLoading
+  ButtonWithLoading,
+  ExternalLink
 } from 'loot-design/src/components/common';
-import { colors, styles } from 'loot-design/src/style';
+import { colors } from 'loot-design/src/style';
 import { signOut, loggedIn } from 'loot-core/src/client/actions/user';
 import { send } from 'loot-core/src/platform/client/fetch';
-import {
-  useBootstrapped,
-  Title,
-  Input,
-  Link,
-  ExternalLink
-} from './subscribe/common';
+import { Title, Input } from './subscribe/common';
+import { useSetThemeColor } from 'loot-design/src/components/hooks';
+import useServerVersion from '../../hooks/useServerVersion';
+
+function Version() {
+  const version = useServerVersion();
+
+  return (
+    <ExternalLink
```

Comment:
Should no longer be a link on the management page, as there's nothing to link to. See [this comment](https://github.com/actualbudget/actual/pull/112#issuecomment-1189780228) from previous pr#112